helm/cassandra/values.yaml

```diff
@@ -16,87 +18,82 @@ image:
+      storage: 1Gi
 
 #sidecarEnv:
 #  - name: GOOGLE_APPLICATION_CREDENTIALS
-#    value: "/tmp/backup-creds/my-service-key.json"
-
-
-#backupSecretVolumeSource:
-#  # example from doc/backup_restore.md
-#  secretName: gcp-auth-reference
-#  #     type is a workaround for https://github.com/kubernetes/kubernetes/issues/68466
-#  type: array
-#  items:
-#    - key: my-service-key.json
-#      path: my-service-key.json
-
-#  racks:
-#    - name: "west1-b"
-#      labels:
-#        failure-domain.beta.kubernetes.io/zone: europe-west1-b
-#    - name: "west1-c"
-#      labels:
-#        failure-domain.beta.kubernetes.io/zone: europe-west1-c
-#    - name: "west1-a"
-#      labels:
-#        failure-domain.beta.kubernetes.io/zone: europe-west1-a#
-
-#Example authentication for AWS S3 based backups
-#env:
+#    value: "/tmp/backup-creds/gcp.json"
 #  - name: AWS_ACCESS_KEY_ID
 #    valueFrom:
 #      secretKeyRef:
-#        name: awsbackuptest
-#        key: access
+#        name: backup-secrets
+#        key: awsaccesskeyid
 #  - name: AWS_SECRET_ACCESS_KEY
 #    valueFrom:
 #      secretKeyRef:
-#        name: awsbackuptest
-#        key: secret
+#        name: backup-secrets
+#        key: awssecretaccesskey
 #  - name: AWS_REGION
-#    value: "us-west-2"
+#    valueFrom:
+#      secretKeyRef:
+#        name: backup-secrets
+#        key: awsregion
+#  - name: AZURE_STORAGE_ACCOUNT
+#    valueFrom:
+#      secretKeyRef:
+#        name: backup-secrets
+#        key: azurestorageaccount
+#  - name: AZURE_STORAGE_KEY
+#    valueFrom:
+#      secretKeyRef:
+#        name: backup-secrets
+#        key: azurestoragekey
 
-#Example authentication for AWS GCP based backups
-#env:
-#  - name: GOOGLE_APPLICATION_CREDENTIALS
-#    value: "/tmp/user-config/service_key.json"
-#  - name: GOOGLE_CLOUD_PROJECT
-#    value: "cassandra-operator"
-#  - name: BUCKET_NAME
-#    value: "my-bucket"
-#
-#userConfigMapVolumeSource:
-#  - name: configMapReference
+# env properties for Cassandra container, similar to sidecarEnv
+# cassandraEnv:
 
-## Example GCP ConfigMap
-#apiVersion: extensions/v1beta1
-#kind: ConfigMap
-#metadata:
-#  name: configMapReference
-#data:
-#  service_key.json: "XXXXXXXXX"
+#backupSecretVolumeSource:
+#  secretName: gcp-auth-reference
+#  #   type is a workaround for https://github.com/kubernetes/kubernetes/issues/68466
+#  type: array
+#  items:
+#    - key: gcp.json
+#      path: gcp.json
+
+#racks:
+#  - name: "west1-b"
+#    labels:
+#      failure-domain.beta.kubernetes.io/zone: europe-west1-b
+#  - name: "west1-c"
+#    labels:
+#      failure-domain.beta.kubernetes.io/zone: europe-west1-c
+#  - name: "west1-a"
+#    labels:
+#      failure-domain.beta.kubernetes.io/zone: europe-west1-a
 
+#userConfigMapVolumeSource:
+#  - name: configMapReference
 
+#restoreFromBackup: backup-name
 
-#restoreFromBackup: backup-hostname
+prometheusEnabled: false
 
-prometheusEnabled: false
\ No newline at end of file
+#serviceAccountName: cassandra
```

Comment:
@johananl should not this be enabled? wdyt? I think when Cassandra operator is deployed, it will deploy all service accounts and roles along with that so this should be fine to have not commented out, right?

Reply:
I think it is probably safer to explicitly use that SA, yes. On most k8s clusters things will probably work also with the default SA (now that we don't run as root), however on *some* clusters with tighter PSPs it will break. So yes.